spec/system/pages/home_spec.rb

```diff
@@ -111,7 +111,6 @@
   context "with Elasticsearch enabled", :elasticsearch do
     before(:each) do |example|
       stub_feature_flag(:use_elasticsearch_for_search, true)
-      stub_feature_flag(:show_recently_added_section, true)
```

Comment:
@jenlu33 there is another test in this file that uses the flag in line 282. Can we also remove that one?